### PR TITLE
DISTMYSQL-138: MySQL Orchestrator does not support new semi-sync variables introduced in MySQL 8.0.26

### PR DESCRIPTION
**Pull request by kamil-holubicki**
https://jira.percona.com/browse/DISTMYSQL-138

Problem:
MySql 8.0.26 introduced new semisync replication plugins
semisync_source.so and semisync_replica.so
in place of
semisync_master.so and semisync_slave.so
All semisync replication plugins' variables were renamed in the way
that 'master' was replaced with the term 'source' and 'slave' was
replaced by 'replica'.

As 8.0.26 contains also old plugins and when they are installed
everything works fine it is not enough to detect the server version.
Moreover, it is possible to mix the old source plugin with the new replica
plugin and the opposite.

Solution:
Detect which plugins are installed and use proper variable names.

Please note that for now, this PR contains 2 commits as it is based on work done in the scope of DISTMYSQL-140 branch, which has open PR to master. We need to do this first to spin-up dev environment. Once DISTMYSQL-140 is done and that part is merged, this PR will contain only the part which is in the scope of this ticket.
In other words, I would like to know your opinion about commit 5823e608a19c9a8717229744fcbce9cdf30e39fa

### CODE REVIEW

File: go/inst/instance_dao.go [hunk 1]
```diff
@@ -486,16 +530,39 @@ func ReadTopologyInstanceBufferable(instanceKey *InstanceKey, bufferWrites bool,
...
+					if matched {
+						instance.SemiSyncReplicaStatus = (m.GetString("Value") == "ON")
+					}
 					return nil
 				})
+				/*
+					log.Debugf("%v, SemiSyncMasterStatus: %v", instance.Key, instance.SemiSyncMasterStatus)
+					log.Debugf("%v, SemiSyncMasterClients: %v", instance.Key, instance.SemiSyncMasterClients)
+					log.Debugf("%v, SemiSyncReplicaStatus: %v", instance.Key, instance.SemiSyncReplicaStatus)
+				*/
```
Comment (egegunes) [inline go/inst/instance_dao.go:565]: Apart from these debug logs, LGTM.

It's definitely looks more clear than what I imagined initially 👍

**Review by hors — APPROVED** [review]

**Review by egegunes — APPROVED** [review]
